README.md

```diff
@@ -1,25 +1,33 @@
 # Package Storage
-This is a storage repository for the packages served through the [package registry](https://github.com/elastic/package-registry) service.
+This is a storage repository for the packages served through the [package registry](https://github.com/elastic/package-registry) service.  See the package-registry repo for basic registry api usage and examples.
 
-It contains 3 branches with the packages for the different environments:
+Package storage repo contains 3 branches with the packages for the different environments:
 
 * snapshot
 * staging
 * production
 
 Here's how these branches relate to repositories and other aspects of packages.
 
-|                       | Snapshot                    | Staging                    | Production                |
-|-------------------    |-------------------------    |------------------------    |-----------------------    |
-| URL                   | epr-snapshot.elastic.co     | epr-staging.elastic.co     | epr.elastic.co            |
-| Add package           | Commit                      | Commit                     | PR                        |
-| Version overwrite     | yes                         | if needed                  | no                        |
-| Stack Version         | *-SNAPSHOT                  | *-SNAPSHOT                 | Released (candidates)     |
-| Registry Version      | fixed version                | Stable release             | Stable release            |
-| Branch                | snapshot                    | staging                    | production                |
-| Packages              | snapshot+staging+prod       | staging+production         | production                |
-| Release               | Manual                      | Manual                  | Manual                 |
-| Docker image          | snapshot                    | staging                    | production                |
+|                         | Snapshot                    | Staging                    | Production                |
+|-------------------      |-------------------------    |------------------------    |-----------------------    |
+| URL                     |  [epr-snapshot.elastic.co](https://epr-snapshot.elastic.co/search) | [epr-staging.elastic.co](https://epr-staging.elastic.co/search) | [epr.elastic.co](https://epr.elastic.co/search) |
```

Comment:
Do we want to link to the `/search` endpoints or just the root endpoint (`/`)?

Reply:
I wanted /search to help as much as possible for folks to see some results.